Basic-Car-Maintenance/Shared/Dashboard/ViewModels/DashboardViewModel.swift

```diff
@@ -9,6 +9,12 @@ import FirebaseFirestore
 import FirebaseFirestoreSwift
 import Foundation
 
+enum LoadingState: String {
+    case isLoading
+    case loaded
+    case error
```

Comment:
I don't think an `error` state is needed, this would indicate that the data has finished loading. What I was thinking was an `isLoading` boolean value